tests/snapshots/parse_file__imp_map.hvm.snap

```diff
@@ -8,4 +8,40 @@ input_file: tests/golden_tests/parse_file/imp_map.hvm
 
 (Map/set map key value) = match map = map { Map/node: switch _ = (== 0 key) { 0: switch _ = (% key 2) { 0: (Map/node map.value (Map/set map.left (/ key 2) value) map.right); _ _-1: (Map/node map.value map.left (Map/set map.right (/ key 2) value)); }; _ _-1: (Map/node value map.left map.right); }; Map/leaf: switch _ = (== 0 key) { 0: switch _ = (% key 2) { 0: (Map/node * (Map/set Map/leaf (/ key 2) value) Map/leaf); _ _-1: (Map/node * Map/leaf (Map/set Map/leaf (/ key 2) value)); }; _ _-1: (Map/node value Map/leaf Map/leaf); }; }
 
+(STRING_NIL_TAG) = 0
```

Comment:
Could you add a `ctx.prune` pass at the end of this test? Just so that it doesn't keep changing everytime we add something to the builtins